doc/domains.rst

```diff
@@ -76,7 +76,7 @@ directive name.
 
 .. rubric:: Default Domain
 
-To avoid having to writing the domain name all the time when you e.g. only
+To avoid having to write the domain name all the time if you only
 describe Python objects, a default domain can be selected with either the config
```

Comment:
Hmmm... I think this might be more clear:

Suppose that a project's documentation only describes Python objects. Specifying a default domain allows documentation authors to avoid having to write the domain name for every directive, role, etc. A default domain can be selected with either the config value :confval:`primary_domain` or this directive:

...